### PR TITLE
Use condor_ce_config_val on htcondor-ce (SOFTWARE-2629)

### PR DESCRIPTION
**Pull request by sthapa**

### CODE REVIEW

File: condor/condor_meter [hunk 1]
```diff
@@ -774,7 +774,11 @@ def htcondor_configured():
     """ 
     try:
         path = GratiaCore.Config.getConfigAttribute("CondorLocation")
-        condor_config_path = os.path.join(path, "bin", "condor_config_val")
+        if 'htcondor-ce' in g_probe_config and g_probe_config is not None:
```
Comment (brianhlin) [inline condor/condor_meter]: Can you change this to `g_probe_config.get('htcondor-ce')`?

File: condor/condor_meter [hunk 2]
```diff
@@ -774,7 +774,11 @@ def htcondor_configured():
...
+          condor_binary = 'condor_ce_config_val'
```
Comment (brianhlin) [inline condor/condor_meter]: `condor_config_binary` would be more descriptive

File: condor/condor_meter (hunk 1 above)
Comment (brianhlin) [reply condor/condor_meter]: Scratch that, I misread the statement and thankfully @edquist had a strong opinion about my suggestion. The statement is still broken though, it should read:

`if g_probe_config is not None and 'htcondor-ce' in g_probe_config:`

Comment (sthapa) [reply condor/condor_meter]: Fixed

File: condor/condor_meter (hunk 2 above)
Comment (sthapa) [reply condor/condor_meter]: renamed

**Pull request merged**